util_inference_example.py

```diff
@@ -0,0 +1,87 @@
+"""
+Usage: python util_inference_example.py image annotations
+
+Adapted from the dense_inference.py to demonstate the usage of the util
+functions.
+"""
+
+import sys
+import numpy as np
+import cv2
+import densecrf as dcrf
+import matplotlib.pylab as plt
+from skimage.segmentation import relabel_sequential
+
+from utils import compute_unary, create_pairwise_bilateral, \
+    create_pairwise_gaussian
```

Comment:
I believe I need to move things around to achieve this: make `densecrf` a python package which imports the Cython-generated library and the utils. So unless someone knows an easier way, I'm OK merging this (after you address the other minor comments) and doing it myself.